tobaccocessation/templates/main/index.html

```diff
@@ -59,7 +59,7 @@
                     <h4>Course Materials</h4>
                     <ul>
                         <li>&nbsp;&nbsp;
-                            <a href="http://ccnmtl.columbia.edu/tobaccocessation/TobaccoCessation_CourseGuide_{{profile.role|capfirst}}.pdf" target="_blank" class="btn">Download Course Guide&nbsp;<i class="icon-download"></i>
+                            <a href="https://www1.columbia.edu/sec/ccnmtl/remote/static/tobaccocessation/TobaccoCessation_CourseGuide_{{profile.role|capfirst}}.pdf" target="_blank" class="btn">Download Course Guide&nbsp;<i class="icon-download"></i>
```

Comment:
Oh - I see. In tobacco, there is no "Student" role. There's Main, General, Perio & Surgery. So this works properly -- http://ccnmtl.columbia.edu/tobaccocessation/TobaccoCessation_CourseGuide_Main.pdf
as does this  https://www1.columbia.edu/sec/ccnmtl/remote/static/tobaccocessation/TobaccoCessation_CourseGuide_Main.pdf